cmd/diff/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/pkg/diff"
+)
+
+var (
+	color       = flag.Bool("color", false, "color")
+	contextSize = flag.Int("context", 3, "context size")
+)
+
+func fileLines(filename string) ([]string, error) {
+	f, err := os.Open(filename)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	var lines []string
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		lines = append(lines, s.Text())
+	}
+	return lines, s.Err()
+}
+
+func run() error {
+	ctx := context.Background()
```

Comment:
Let's either inline this in the `diff.Myers` call or place it in the line or two immediately preceding it. (And possibly add a timeout flag and construct a deadline context from it?)

Reply:
Done.